qt/python/mantidqt/widgets/matrixworkspacedisplay/table_view_model.py

```diff
@@ -0,0 +1,202 @@
+# coding=utf-8
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench.
+#
+#
+from __future__ import (absolute_import, division, print_function)
+
+from qtpy import QtGui
+from qtpy.QtCore import QVariant, Qt, QAbstractTableModel
+
+
+def enum(**enums):
```

Comment:
Can this use the `Enum` type defined in [mantidqt/py3compat/enum.py)](https://github.com/mantidproject/mantid/blob/master/qt/python/mantidqt/py3compat/enum.py)?

Reply:
I didn't know we had an enum impl! Thank you, I will definitely use it

Reply:
We needed on in a few places and we can't use the enum from Python 3 solely.